test/install/test_no_matplotlib.py

```diff
@@ -4,12 +4,17 @@
 import numpy as np
 import pytest
 
+from sklearn.base import BaseEstimator, ClassifierMixin
+
 from fairlearn.postprocessing import ThresholdOptimizer, plot_threshold_optimizer
 from fairlearn.postprocessing._threshold_optimizer import _SUPPORTED_CONSTRAINTS
 from fairlearn.postprocessing._constants import _MATPLOTLIB_IMPORT_ERROR_MESSAGE
 
 
-class FakePredictor:
+class FakePredictor(BaseEstimator, ClassifierMixin):
```

Comment:
nit: should this be FakeEstimator, instead of FakePredictor? I'm going purely by sklearn definitions here.

Reply:
I didn't want to touch the tests much. I'm not happy with them and I think they can use some restructuring in regard to these tiny estimators. But I rather leave that to another PR.

Reply:
I'm very open to suggestions on all of these, but that's probably a discussion for "issues" or at least elsewhere.